core/src/test/java/org/infinispan/api/ConditionalOperationPrimaryOwnerFailTest.java

```diff
@@ -70,7 +70,6 @@ public void testEntryNotWrapped() throws Throwable {
          log.debugf("wrapEntryForWriting invoked with %s", context);
 
          Object mvccEntry = invocation.callRealMethod();
-         assertNull(mvccEntry, "Entry should not be wrapped!");
```

Comment:
I see now, you need to return 

`invocation.callRealMethod().thenRun(() -> assertNull(...))`

The `mvccEntry` variable was always `null` because the method was `void`.

Reply:
I probably should join it, no?